### PR TITLE
Answer: 1

### PR DESCRIPTION
**Pull request by dmmishchenko**
## Checklist for challenge submission

- [x] Start your PR message with Answer:${challenge_number}

## Link to challenge:
[🟢 Projection](https://angular-challenges.vercel.app/challenges/angular/1-projection/)

## Image:
Screenshot 2023-10-29 at 12 22 54

### CODE REVIEW

File: apps/angular/projection/src/app/component/city-card/city-card.component.ts [hunk 1]
```diff
@@ -1,13 +1,50 @@
...
-  constructor() {}
+export class CityCardComponent implements OnInit, CardListInterface {
+  cities: City[] = [];
+  constructor(private http: FakeHttpService, private store: CityStore) {}
 
-  ngOnInit(): void {}
+  ngOnInit(): void {
+    this.http.fetchCities$.subscribe((s) => this.store.addAll(s));
+
+    this.store.cities$.subscribe((s) => (this.cities = s));
```
Comment (tomalaforge) [inline apps/angular/projection/src/app/component/city-card/city-card.component.ts:40]: you can use an asyncPipe to avoid imperative coding

File: apps/angular/projection/src/app/ui/list-item/list-item.component.ts [hunk 2]
```diff
@@ -1,35 +1,14 @@
...
 
 @Component({
   selector: 'app-list-item',
   template: `
     <div class="border border-grey-300 py-1 px-2 flex justify-between">
-      {{ name }}
-      <button (click)="delete(id)">
-        <img class="h-5" src="assets/svg/trash.svg" />
-      </button>
+      <ng-content></ng-content>
```
Comment (tomalaforge) [inline apps/angular/projection/src/app/ui/list-item/list-item.component.ts:8]: Why did you delete the trash button from this component ?

**Comment by tomalaforge** [comment]
Nice work and nice use of `ngTemplateOutlet` and `ngContent`

File: apps/angular/projection/src/app/ui/list-item/list-item.component.ts (hunk 2 above)
Comment (dmmishchenko) [reply apps/angular/projection/src/app/ui/list-item/list-item.component.ts:8]: Oh, sorry should i return it?
I did it because in my approach helps to handle delete inside card component

File: apps/angular/projection/src/app/component/city-card/city-card.component.ts (hunk 1 above)
Comment (dmmishchenko) [reply apps/angular/projection/src/app/component/city-card/city-card.component.ts]: changed to asyncPipe, thanks for the feedback
